torch/_dynamo/variables/builtin.py

```diff
@@ -1201,6 +1201,12 @@ def _call_min_max_binary(self, tx: "InstructionTranslator", a, b):
                 "call_function", fn, *proxy_args_kwargs([a, b], {})
             )
             return SymNodeVariable.create(tx, proxy, None)
+        elif isinstance(a, ConstantVariable) and isinstance(b, ConstantVariable):
+            value = self.fn(
+                a.as_python_constant(),
+                b.as_python_constant(),
```

Comment:
Do we need to do a.guard_as_python_constant or does Dynamo already guard on ConstantVariables?

Reply:
Correct me if I'm wrong here. But since they are constants in the source code, we would only have different constants if one change the source code, no?

```python
def test_map_max_const(a):
    return max(map(lambda x: x, [1, 2, 3]))
```

If so, having guards matters in this case?